libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -76,6 +77,9 @@ def __init__(self):
         self.control_nodeport_https = os.getenv("CONTROL_HTTPS_PORT",
                                                 self.deploy_cfg["control_port_https"])
         self.test_dir_path = os.path.join(TEST_DATA_FOLDER, "testDeployment")
+        self.data_only_list = ["data-only", "standard"]
```

Comment:
Do you not want to get this from config?

Reply:
these are static entires we will not read from config